v3/plugins/redhat/vscode-camelk/Contributing.md

```diff
@@ -0,0 +1,4 @@
+The VS Code Tooling for Camel K extension requires VS Code Kubernetes to be installed.
+It comes with a specific sidecar image. This sidecar image can be found in this repo https://github.com/che-dockerfiles/che-sidecar-camelk .
+
+When updating the Camel K or the Kubernetes extension, the sidecar image might need to be updated too. Please note that the camel K sidecar image relies on the Kubernetes sidecar image.
```

Comment:
This could be a little clearer -- maybe something like
```markdown
Since the sidecar image used for Camel K depends on the [che-sidecar-kubernetes-tooling](https://github.com/che-dockerfiles/che-sidecar-kubernetes-tooling/) image, updating the Camel K extension may require also updating the sidecar image.
```

Reply:
agree, I'm not clear enough. I can see it especially as your proposition doesn't reflect what I wanted say ahah.
I will rework it

Reply:
@apupier I also notice that there's a 1.0.4 image for kubernetes-tooling -- is it incompatible with Camel K?

Reply:
yes.
Work in progress to use it:
https://github.com/che-dockerfiles/che-sidecar-camelk/pull/2
https://github.com/che-dockerfiles/che-sidecar-camelk/pull/1